project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.11")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.12")
```

Comment:
Is there a reason we don't use https://github.com/sbt/sbt-release ? It looks like this is the canonical version now

Reply:
I think it is already the case: https://github.com/sbt/sbt-release#usage (and I'm relieved because that would have been painful to migrate :) )